tests/model/issue.model.spec.ts

```diff
@@ -43,3 +48,98 @@ describe('Issue model class', () => {
         });
     });
 });
+
+describe('Issue', () => {
+    const dummyTeam = new Team({
+        id: 'F09-2',
+        teamMembers: []
+    });
+    const dummyIssue = Issue.createPhaseBugReportingIssue(ISSUE_WITH_EMPTY_DESCRIPTION);
+    const otherDummyIssue = Issue.createPhaseBugReportingIssue(ISSUE_WITH_ASSIGNEES);
+    const dummyIssueWithTeam = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_EMPTY_DESCRIPTION, dummyTeam);
+
+    const noReportedDescriptionString = 'No details provided by bug reporter.\n';
+    const tutorResponseStringHeader = '# Tutor Moderation\n\n';
+
+    const newIssueDispute = new IssueDispute('Cannot Work', 'Help Please');
+    const newTesterResponse = new TesterResponse('Cannot Work', 'Help Please', '- [ ] Not Done', 'Reason');
+
+    it('.clone() should intialise the cloned issue with the correct phase and team', () => {
+        const phaseBugReportingIssue = dummyIssue.clone(Phase.phaseBugReporting);
+        expect(phaseBugReportingIssue).toEqual(dummyIssue);
+
+        const phaseTeamResponseIssue = dummyIssueWithTeam.clone(Phase.phaseTeamResponse);
+        expect(phaseTeamResponseIssue.githubComments).toEqual(dummyIssueWithTeam.githubComments);
+        expect(phaseTeamResponseIssue.teamAssigned).toEqual(dummyTeam);
+
+        const phaseTesterResponseIssue = dummyIssueWithTeam.clone(Phase.phaseTesterResponse);
+        expect(phaseTesterResponseIssue.githubComments).toEqual(dummyIssueWithTeam.githubComments);
+
+        const phaseModerationIssue = dummyIssueWithTeam.clone(Phase.phaseModeration);
+        expect(phaseModerationIssue.githubComments).toEqual(dummyIssueWithTeam.githubComments);
+        expect(phaseModerationIssue.teamAssigned).toEqual(dummyTeam);
+    });
+
+    it('.createGithubIssueDescription() forms the correct GitHub Issue description for the issue', () => {
+        const phaseBugReportingIssue = dummyIssue.clone(Phase.phaseBugReporting);
+        expect(phaseBugReportingIssue.createGithubIssueDescription()).toEqual(noReportedDescriptionString);
+
+        const phaseBugReportingIssueWithDescription = otherDummyIssue.clone(Phase.phaseBugReporting);
+        expect(phaseBugReportingIssueWithDescription.createGithubIssueDescription())
+            .toEqual(`${phaseBugReportingIssueWithDescription.description}\n`);
+    });
+
+    it('.clone() should be able to get the proper Team Response', () => {
```

Comment:
Shouldn't this be `.createGithubTeamResponse()` and not clone?
(Same for the other tests, besides the test for clone)